packages/client/wallets/aa/src/blockchain/wallets/EVMSmartWallet.ts

```diff
@@ -111,7 +111,7 @@ export class EVMSmartWallet extends LoggerWrapper {
                 const { request } = await client.simulateContract(tx);
                 return client.writeContract(request);
             } catch (error) {
-                logError("[TRANSFER] - ERROR_TRANSFERRING_TOKEN", {
+                logger.logError("[TRANSFER] - ERROR_TRANSFERRING_TOKEN", {
```

Comment:
and then with that change, I'd expect `LoggerWrapper` to have a property:
```
logger: Logger = LoggerProvider.getLogger()
```
so that this code stays, but without the global var 
`import { logger } from "@/services/logging";`